### DISCUSSION TITLE
reference Ecosystems introuvable ?

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#525] [General]
Bonjour,
Je recherche les données , tables liées aux champs "Responsable ", "Réferent fonctionnel" et "Editeur" des Applications.
je ne retrouve aucun table  correspondante dans la vue écosystème , voir les autres vues ?
Pourriez vous me dire ou les trouver ?
merci

**Comment by moiiom59** [comment]
J'ai trouve la reponse ! il s'agit de liste libre . 
Parcontre pourquoi le champs Editeur et Exploitation ne font pas référence a la même liste ? C'est dommage de propose la liste Editeur en format libre alors que généralement c'est l'exploitant ou un prestaire ou la DSI ? L'editeur pourrait avoir sa propre base ou faire réference a la table Entité.
merci

**Comment by dbarzin** [comment]
Les éditeurs ont une base de référence qui est basée sur les CPE (Common Platform Enumeration).
Avez-vous installé la base des CPE comme décrit ici :
https://github.com/dbarzin/mercator/blob/master/INSTALL.fr.md#importer-la-base-de-donn%C3%A9es-des-cpe

**Comment by moiiom59** [comment]
Bonjour,
Dans tous les cas on ne retrouvera pas les éditeurs des progiciels de santé ! Il serait plus judicieux de pointé sur la table Entité pour les type d’entité ‘Type_entity’ qui serait égale à ‘Exploitation’ pour le champ Exploitation et égale à ‘Editeur’ pour le champs Editeur

Dans le formulaire Application, le champs Editeur dans la zone écosystème reste libre, alors que je suis d’accord  CPE Editeur fait bien référence a la liste des éditeur de la base CPE

Et justement dans le formulaire Entité le champs Type d’entité auquel je fait référence ne permet pas d’ajouter d’information ?

Votre produit est vraiment très pratique que l’on espère qu’il s’adapte parfaitement à nos organisations
Merci à vous


Bien cordialement,

**Comment by dbarzin** [comment]
La base des PCE ne contient effectivement pas (pour le moment) la base des éditeurs des progiciels de santé, c'est pour cela qu'il est possible de l'ajouter manuellement dans la liste déroulante des CPE.

Pour le lien avec le champ éditeur et la table entité, votre raisonnement est tout à fait logique. Ce point va être ajouter à la feuille de route.

Merci,

**Comment by moiiom59** [comment]
Dans le formulaire Application nous avons le champ Editeur qui pourrait faire référence à la table Entité et le champs CPE-Editeur qui fera référence à la table CPE, c’est bien ça ?

Merci à vous

Bien cordialement,

**Comment by dbarzin** [comment]
Oui, c'est cela l'idée.

**Comment by dbarzin** [comment]
Je reviens sur l'idée de définir une entité pour chaque éditeur. 
C’est souvent avec l’exploitant que l’organisation a une relation. Il n'existe pas de relation entre l’organisation et l’éditeur.  C’est l’exploitant que l’organisation va contacter en cas d’incident ou de problème technique. L’organisation n’a parfois même pas de contact auprès de l’éditeur à moins que l’éditeur ne soit lui-même l’exploitant.
On risque donc de définir des entités sans relations avec l’organisation. Je me demande si c’est utile ?

**Comment by moiiom59** [comment]
Bonjour monsieur Barzan,
En effet , il faut juste que les 2 champs Exploitation et Editeur de la table application pointe sur la table Entités (ce qui n’est pas le cas aujourd’hui) . Ceci pour retrouver dans Entités l’ensemble des contacts lié à l’applicatif identifiés de façon unique.
L’exploitant n’ayant parfois aucune relation avec le fournisseur.

Bien cordialement,
